### PR TITLE
DRA-1412 HERO update

### PR DESCRIPTION
**Pull request by danielqvistcom**
Pkt. 1 #HERO update. Left/right coll. styling + font-updates.

### CODE REVIEW

**Comment by jesperlauridsen** [comment]
On a small screen
image
the button font now overflows the container.

File: src/components/search/Hero.vue [hunk 1]
```diff
@@ -436,7 +475,14 @@ h1 {
 		flex-direction: row;
 		max-width: 1150px;
 		justify-content: space-between;
-		align-items: center;
+	}
+	.container h1 {
+		margin-top: 105px;
+	}
+	.link-container .link:hover {
```
Comment (jesperlauridsen) [inline src/components/search/Hero.vue:482]: Is there a reason for not putting a "transition: all 0.25s linear 0s" on this, to make the transition smoother?

Comment (jesperlauridsen) [reply src/components/search/Hero.vue:482]: On this note, if you wanna animate the underline too, it should probably be a :after effect instead. text-decoration:underline has no animation features, so it just pops in and out visually.

Comment (danielqvistcom) [reply src/components/search/Hero.vue:486]: You're right. Much smoother with the transition.
When the animation is this fast, I think it's ok with the "underline" effekt.

**Pull request merged**